.github/workflows/ci_tests.yml

```diff
@@ -56,6 +56,14 @@ jobs:
           distribution: temurin
           cache: gradle
 
+      - name: Pull genai-common
```

Comment:
Since we pull from `google()` before `mavenLocal()`, it will get overruled by the published version

https://github.com/firebase/firebase-android-sdk/blob/ade33698703c50cdadff6a97fc27e2ee5eaab625/build.gradle#L75C8-L75C17

see comment below